jump-leveleditor/src/main/java/com/bitdecay/jump/leveleditor/utils/LevelUtilities.java

```diff
@@ -17,10 +17,13 @@ public static Level loadLevel(String fileName) {
 	}
 
 	public static Level loadLevel(File file) {
-
-		return null;
+		return FileUtils.loadFileAs(Level.class, file);
 	}
 
+	public static Level loadLevelFromJson(String json){
+        return FileUtils.loadFileAs(Level.class, json);
```

Comment:
So the reason this has to be here is that from within the game, when I load a level, I need to be able to just supply the json myself.  That way I can make a call to either Gdx.files.classpath or use the ClassLoader.getResource method.